samples/python/console/speech_synthesis_auto_style_sample.py

```diff
@@ -0,0 +1,96 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# Copyright (c) Microsoft. All rights reserved.
+# Licensed under the MIT license. See LICENSE.md file in the project root for full license information.
+"""
+Speech synthesis sample with style auto prediction from GPT
+"""
+
+try:
+    import azure.cognitiveservices.speech as speechsdk
+except ImportError:
+    print("""
+    Importing the Speech SDK for Python failed.
+    Refer to
+    https://docs.microsoft.com/azure/cognitive-services/speech-service/quickstart-text-to-speech-python for
+    installation instructions.
+    """)
+    import sys
+    sys.exit(1)
+
+import json
+import os
+import openai
+openai.api_type = "azure"
+openai.api_base = os.getenv("AZURE_OPENAI_ENDPOINT")
+openai.api_version = "2023-05-15"
+openai.api_key = os.getenv("AZURE_OPENAI_KEY")
+
+STYLE_PROMPT = """You are requested to split the following input into sentences and tag a correct style from {0} and `general`.
+Please response with json format, e.g., [{{"sentence": text, "style": style }}]
+Input:  {1}"""
+
+SPEECH_REGION = os.getenv("SPEECH_REGION")
+SPEECH_KEY = os.getenv("SPEECH_KEY")
+
+# Set the voice you want to use
+VOICE = "en-US-JennyNeural"
+
+# paste or read the text you want to synthesize
```

Comment:
Shold you add a comment with URL pointing to the source of this text, like you've done in another sample where you've used text from Gatsby?

Reply:
thanks for the comments, I added it.